plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/server/N4JSLanguageServerImpl.java

```diff
@@ -46,6 +49,25 @@ public void registerExtensions(HeadlessExtensionRegistrationHelper helper) {
 		helper.registerExtensions();
 	}
 
+	/**
+	 * Call this method after calling {@link #initialized(InitializedParams)}
+	 * <p>
+	 * TODO: Fix this in Xtext
```

Comment:
maybe create ticket and add ticket ID here

Reply:
Yes, but I want to discuss this first with @szarnekow.